best-practices/native/react-native/expo/certificates/README.md

```diff
@@ -0,0 +1,42 @@
+# App Store Credentials (Expo)
+
+Both the Google Play store and the Apple store have their own form of unique credentials when it comes to being able to create and submit builds. Expo does a good job of handling these for you, especially upon first build. Sometimes, these will expire, and then some knowledge is required to be able to fix your credentials so you can create a build and start updating again.
+
+## What are these credentials?
+
+### iOS Credentials
+
+Apple has three different credentials that need to be managed: The Distribution Certificate, the Provisioning Profile, and the Push notification certificate. Expo does a good write up on these [on their App Signing Site](https://docs.expo.io/versions/latest/distribution/app-signing/) for initial study on what these do.
```

Comment:
Just would like to mention we there is Development certificate as well.

Reply:
I used two types of Provisioning Profile: 1- Distribution. 2- Development.

Reply:
Is that different from the Distribution cert? I know the Distribution one is tied to the developer as opposed to the project, which the Provisioning profile is tied to.

Reply:
Just looked it up, you're right! I'll make a note of the different types. Thanks @Basim-Elayan !

Reply:
Actually those two different things as far as I know, you could give someone certificate to do development and only that, and you could give someone permission to distribute IOS app without giving ability to do development and so on.

Reply:
I did some reading up on it and modified the paragraph to explain the differences. Let me know what you think.

Reply:
I needed and used provisioning profile for two purposes:
1-  To tie developers and devices to an authorized iPhone Development Team and enables a device to be used for testing (Development -> iOS App Development).
2- Create a distribution provisioning profile to submit my app to the App Store. (Distribution -> App Store).

Reply:
Yeah thats what I read, although [the article I read](https://www.sharpmobilecode.com/making-sense-of-ios-provisioning/) labeled it as "Production" type instead of "Distribution" type. Expo only grabs production ones, so I noted the difference in the latest change.

Reply:
Great Job @Fleury14, Thanks for sharing these info with us.